src/SFA.DAS.EmployerAccounts/DependencyResolution/AccountApiClientRegistry.cs

```diff
@@ -0,0 +1,16 @@
+﻿using SFA.DAS.EAS.Account.Api.Client;
+using StructureMap;
+
+namespace SFA.DAS.EmployerAccounts.DependencyResolution
+{
+    public class AccountApiClientRegistry : Registry
```

Comment:
I've added this registry here, rather than in the client project, so that we don't introduce a dependency on StructureMap in the client.